apps/testapp/package.json

```diff
@@ -12,7 +12,7 @@
     "@chakra-ui/react": "^2.8.0",
     "@coinbase/wallet-sdk-3.7.2": "npm:@coinbase/wallet-sdk@3.7.2",
     "@coinbase/wallet-sdk-3.9.3": "npm:@coinbase/wallet-sdk@3.9.3",
-    "@coinbase/wallet-sdk-4.0.1": "npm:@coinbase/wallet-sdk@4.0.1",
+    "@coinbase/wallet-sdk-4.0.0": "npm:@coinbase/wallet-sdk@4.0.0",
```

Comment:
should this be 4.0.2?

Reply:
think @bangtoven mentioned pg has some issue w 4.0.1, think we are going to revert back to 4.0.0 for now, will look into this later when we released 4.0.2

Reply:
yeah, had to revert playground 4.0.1 bump https://github.com/coinbase/coinbase-wallet-sdk/pull/1302

Reply:
what was the issue?

Reply:
@nateReiners 
quite weird. some gnarly issue with yarn.lock file i guess.
i wonder why it didn't get caught in ci 🤔